### PR TITLE
week1 homework finished

### PR DESCRIPTION
**Pull request by dhemir17**
Review

### CODE REVIEW

**Comment by dhemir17** [comment]
Homework done

**Comment by dhemir17** [comment]
review

File: Week1/homework/exercise1.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+console.log("Hello World"); //English
+console.log("Merhaba Dunya"); //Turkish
+console.log("Hallo wereld"); //Dutch
+console.log("Hallo Welt"); //German
+console.log("Pozdrav svijetu"); //Bosnia
+console.log("Pershendetje bote"); //Albania
+console.log("Salam dünya"); //Azerbajcan
+console.log("Bonjour tout le monde"); //French
+console.log("Ahoj svět"); //Checzka
+console.log("Привет мир"); //Russian
```
Comment (burakcoskun) [inline Week1/homework/exercise1.js:12]: looks good

File: Week1/homework/exercise10.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+console.log("The length of array animals is: " + animals.length);
+
+if (pets.length == animals.length) {
+    console.log("They are the same!");
+
+}
+
+else {
+    console.log("Two different sizes");
+}
```
Comment (burakcoskun) [inline Week1/homework/exercise10.js:15]: nice

File: Week1/homework/exercise2.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+"use strict";
+console.log('I\'m awesome');
```
Comment (burakcoskun) [inline Week1/homework/exercise2.js:2]: good

File: Week1/homework/exercise3.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+"use strict";
+var numberX;
+console.log("I think x is undefined");
+console.log(numberX);
+numberX = 11;
+console.log("I think the value of x is 11");
+console.log(numberX);
```
Comment (burakcoskun) [inline Week1/homework/exercise3.js:7]: looks good

File: Week1/homework/exercise4.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+"use strict";
+var myString = "Burak Sabah";
+console.log("The value of string is my name and surname");
+console.log(myString);
+myString = "dhemir"
+console.log("Give my nickname to the string");
+console.log(myString);
```
Comment (burakcoskun) [inline Week1/homework/exercise4.js:7]: looks good

File: Week1/homework/exercise5.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+var z = 7.25;
+console.log(z);
+var a = Math.round(z);
+var max;
+if (a > z) {
+    max = a;
+} else {
+    max = z;
+}
+console.log(max);
```
Comment (burakcoskun) [inline Week1/homework/exercise5.js:11]: nice, good that you haven't used Math.max but implemented it yourself

File: Week1/homework/exercise6.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+"use strict";
+var animals = [];
+console.log("I defined an empty array");
+console.log(animals);
+var myFavAnimals = ["tiger", "wolf", "shark"];
+console.log(myFavAnimals);
+myFavAnimals.push("Piglet");
+console.log(myFavAnimals);
```
Comment (burakcoskun) [inline Week1/homework/exercise6.js:8]: nice

File: Week1/homework/exercise7.js [hunk 8]
```diff
@@ -0,0 +1,4 @@
+"use strict";
+
+var mySentence = "Programming is so interesting!";
+console.log(mySentence.length);
```
Comment (burakcoskun) [inline Week1/homework/exercise7.js:4]: nice

File: Week1/homework/exercise8.js [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+if(typeof b == typeof person1) {
+    console.log("SAME TYPE");
+}
+else {console.log("Not the same...");}
+
+if(typeof b == typeof person2) {
+    console.log("SAME TYPE");
+}
+else {console.log("Not the same...");}
```
Comment (burakcoskun) [inline Week1/homework/exercise8.js:43]: looks good, comparing everything with everything else

File: Week1/homework/exercise9.js [hunk 10]
```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+let x = 7;
+x = x % 3; //I define x as the remainder of the formula
+console.log(x); // Log the new X value
+let y = 21; //I define y as the remainder of the formula
+console.log(y % 4); // Log the new Y value
+let z = 13; //I define z as the remainder of the formula
```
Comment (burakcoskun) [inline Week1/homework/exercise9.js:8]: looks good to me

**Review by burakcoskun — APPROVED** [review]
overall very good homework, thanks

**Pull request closed without merging**